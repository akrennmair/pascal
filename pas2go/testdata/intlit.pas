program test;

begin
    writeln('hello: ', -50 - (-14))
end.