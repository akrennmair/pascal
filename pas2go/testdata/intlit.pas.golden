package main

import (
	"github.com/akrennmair/pascal/pas2go/system"
)

var _ = system.Write

// program test
func main() {
	system.Writeln("hello: ", (-50)-(-14))
}
